### PR TITLE
Add deployment manifests & update Arduino code to point to deployed server

### PR DESCRIPTION
**Pull request by EngHabu**
- [X] Add kustomize deployment manifests
- [X] Update ESP32 code to point to the deployed server
- [X] Update ESP32 code to fix sending requests code
- [X] Update docker build to build and push multi-architecture docker images to work on Raspberry Pi's k8s cluster

### CODE REVIEW

File: arduino/client_esp32.ino [hunk 1]
```diff
@@ -54,11 +54,21 @@ void setup()
...
 }
 
+int x = 1;
+
 void loop()
 {
   // read the state of the pushbutton value:
   buttonState = digitalRead(buttonPin);
 
+  if (x < 10)
```
Comment (EngHabu) [inline arduino/client_esp32.ino:64]: Delete this block before merging

**Comment by EngHabu** [comment]
@romoh, please kindly review my change above....

File: deploy/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+resources:
+  - "namespace.yaml"
+  - "ingress.yaml"
+  - deployment.yaml
+  - service.yaml
+
+# Images that should be used
+images:
+  # FlyteAdmin
```
Comment (romoh) [inline deploy/kustomization.yaml:12]: remove

File: deploy/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - name: fencing # match images with this name
+    newTag: 2c4c69614d2656e28b60db9aba75bebe0452be03 # override the tag
```
Comment (romoh) [inline deploy/kustomization.yaml:14]: why not use latest?

**Review by romoh — APPROVED** [review]

File: deploy/kustomization.yaml (hunk 3 above)
Comment (EngHabu) [reply deploy/kustomization.yaml:14]: Great question! This is a bit more predictable.. allows roll forward and backward and more manageable versioning...

**Pull request merged**